### PR TITLE
ACM-8838 Initial Argo pull create wizard implementation

### PR DESCRIPTION
**Pull request by fxiang1**
https://issues.redhat.com/browse/ACM-8838

- Add Argo pull model create/edit wizard
- Add comparison diagram for Argo push/pull comparisons

image

### CODE REVIEW

**Comment by xiangjingli** [comment]
@fxiang1  I am not the frontend development expert. I will let it go so that QE will not be blocked
@chenz4027  Could you help review it when you have time once you are back.

File: frontend/src/routes/Applications/CreateApplication/CreateApplicationArgoPullModel.test.tsx [hunk 1]
```diff
@@ -0,0 +1,480 @@
...
+        }}
+      >
+        <MemoryRouter initialEntries={[NavigationPath.createApplicationArgoPullModel]}>
+          <Route component={() => <CreateApplicationArgoPullModel />} />
+        </MemoryRouter>
+      </RecoilRoot>
+    )
+  }
+
+  test('can create Argo Application Set with Git', async () => {
```
Comment (chenz4027) [inline frontend/src/routes/Applications/CreateApplication/CreateApplicationArgoPullModel.test.tsx:359]: Great tests!

**Review by chenz4027 — COMMENTED** [review]
LGTM, great work @fxiang1

**Pull request merged**